### DISCUSSION TITLE
when I install the patched file to the electribe, it says invalid file.. can anyone help please?

### DISCUSSION THREAD

**Discussion by MUNKE23** [bangcorrupt/hacktribe#248] [Q&A] [answered]
when I install the patched file to the electribe, it says invalid file..

**Comment by BKLronin** [comment, marked as answer]
Renamed and correct folder?

**Reply by MUNKE23** [reply]
yes 100%

**Reply by MUNKE23** [reply]
KORG/electribe/system and I renamed it to SYSTEM.VSB

**Reply by BKLronin** [reply]
Followed instructions closely? There is also a tutorial jupyther book for this.

**Reply by MUNKE23** [reply]
yes im really not sure what's the problem.. could you please post the link here to it?

**Reply by MUNKE23** [reply]
I got a blue electribe so I don't know if the instructions are different for the synth version

**Reply by enorrmann** [reply]
> I got a blue electribe so I don't know if the instructions are different for the synth version

from how to patch
**If installing to factory synth (grey or blue), use -e flag to also modify file header.**

**Reply by bangcorrupt** [reply]
[FAQ - Error, invalid file](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#error-invalid-file)

**Reply by BKLronin** [reply]
Did you do all this ?
https://github.com/bangcorrupt/hacktribe/wiki/how-to#firmware-patch

**Reply by MUNKE23** [reply]
thanks a lot its working now!
the last thing I need some help with is the factory samples how do I get them back?

**Reply by bangcorrupt** [reply]
[Wiki - Factory Samples](https://github.com/bangcorrupt/hacktribe/wiki/How-To#factory-samples)

**Reply by MUNKE23** [reply]
im really lost with this stuff.. sorry for too many questions

**Reply by enorrmann** [reply]
> im really lost with this stuff.. sorry for too many questions

it's all in the docs, try reading them for a while :)
